app/components/login/LoginForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { PropTypes } from 'react';
+import Formsy from 'formsy-react';
```

Comment:
Was formsy already part of the dependencies?

Reply:
Do you mean in package.json? yes, it was. In fact, I didn't know that library, I used it because the user form was using it.

Reply:
Awesome, thanks -- I had missed it before.